package.json

```diff
@@ -16,58 +16,6 @@
   "workspaces": [
     "packages/*"
   ],
-  "jest-junit": {
-    "outputDirectory": "coverage",
-    "outputName": "junit.xml"
-  },
-  "jest": {
-    "testEnvironment": "node",
-    "workerThreads": false,
-    "runInBand": true,
-    "modulePaths": [
-      "<rootDir>"
-    ],
-    "collectCoverage": true,
-    "setupFilesAfterEnv": [],
-    "transformIgnorePatterns": [
-      "node_modules/(?!@noble)",
-      "node_modules/(?!sinon)"
-    ],
-    "collectCoverageFrom": [
-      "src/**/*.{js,ts}"
-    ],
-    "moduleNameMapper": {
-      "^sinon$": "<rootDir>/node_modules/sinon/pkg/sinon.js",
-      "^uint8arrays$": "<rootDir>/node_modules/uint8arrays/cjs/src",
-      "^uint8arrays/(.*)$": "<rootDir>/node_modules/uint8arrays/cjs/src/$1",
-      "^multiformats/(.*)$": "<rootDir>/node_modules/multiformats/cjs/src/$1",
-      "^rxdb/(.*)$": "<rootDir>/node_modules/rxdb/$1",
-      "^dexie$": "<rootDir>/node_modules/dexie",
-      "^core-js/modules/(.*)$": "<rootDir>/node_modules/core-js/modules/$1"
-    },
-    "moduleFileExtensions": [
-      "js",
-      "ts"
-    ],
-    "coverageReporters": [
-      "json-summary",
-      "lcov",
-      "text"
-    ],
-    "reporters": [
-      "default",
-      "jest-junit"
-    ],
-    "coverageThreshold": {
-      "global": {
-        "branches": 58,
-        "functions": 64,
-        "lines": 74,
-        "statements": 71
-      }
-    },
-    "coverageDirectory": "<rootDir>/coverage"
-  },
```

Comment:
Deleted all this `jest` config? Dunno if there's anything that special in here?

Reply:
nop, its all good u generate the coverage with istambul + in the same folder.